package.json

```diff
@@ -23,7 +23,8 @@
   "author": "skynetlabs",
   "license": "MIT",
   "dependencies": {
+    "@skynetlabs/skynet-nodejs": "https://github.com/SkynetLabs/nodejs-skynet#refs/heads/in-memory-upload",
     "gun": "^0.2020.1235",
-    "skynet-js": "^3.0.2"
+    "skynet-js": "^4.0.27-beta"
```

Comment:
The PR says `remove skynet-js`, wanted to confirm that this is meant to be here. If so, it should probably be 4.1.0.